### PR TITLE
Remove `status:merge-when-green` label after merge

### PR DESCRIPTION
**Pull request by ihostage**
Inspired by https://github.com/playframework/playframework/pulls?q=is%3Apr+is%3Aclosed

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -12,8 +12,17 @@ pull_request_rules:
...
       delete_head_branch: {}
+      label:
+        remove: [ "status:merge-when-green" ]
+
+  - name: Labeling for Scala Steward PR's
+      conditions:
+        - author=scala-steward
+      actions:
+        label:
+          add: [ "type:updates" ]
```
Comment (ignasi35) [inline .mergify.yml:28]: Oh, I love this bit! Thanks @ihostage

**Review by ignasi35 — APPROVED** [review]

**Comment by ignasi35** [comment]
Hmmm, _mergify_  seems to disagree: `The new Mergify configuration is invalid`

File: .mergify.yml (hunk 1 above)
Comment (octonato) [reply .mergify.yml:28]: YAML indentation hitting again!

Comment (octonato) [reply .mergify.yml:28]: @ihostage, there is a mergify lint. Check their docs.

Comment (ihostage) [reply .mergify.yml:28]: Oh, sorry... I copied from another repo and not checked indentation after pasted this block 🤦‍♂️

Comment (ihostage) [reply .mergify.yml]: Should be ok now

**Review by octonato — APPROVED** [review]
Great! Thanks @ihostage

**Pull request merged**